esp/public/media/scripts/program/modules/catalog.js

```diff
@@ -0,0 +1,169 @@
+// Inject difficulty data from the tag to support custom difficulties
+const DIFFICULTIES = INJECTED_DIFFICULTIES !== "None" 
+    ? JSON.parse(INJECTED_DIFFICULTIES)
+    : [
+        ["*", "This Shouldn't Appear!"],
+        ["**", "This Shouldn't Appear!"],
+        ["***", "This Shouldn't Appear!"],
+        ["****", "This Shouldn't Appear!"]
+    ];
+
+const MODIFIED_COLOR = "#0066ff22";
+
+const FILTER_IDS = ["grade_filter", "difficulty_filter", "status_filter", "duration_filter"];
+
+/**
+    Converts from hours to a formatted duration (eg. "0.05" -> "5 mins", "1.5" -> "1 hour 30 mins")
+    @param {string} numString - a duration as a float in a string (eg. "0.05")
+    @returns {string} the formatted time (eg. "1 hour 30 mins")
```

Comment:
```suggestion
    Converts from hours to a formatted duration (eg. "0.05" -> "3 mins", "1.5" -> "1 hour 30 mins")
    @param {string} numString - a duration as a float in a string (eg. "0.05")
    @returns {string} the formatted time (eg. "1 hour 30 mins")
```

Just realized I had made a small typo here before :)